_tools/check_licence.ts

```diff
@@ -32,14 +30,8 @@ for await (
 ) {
   const content = await Deno.readTextFile(path);
   if (!content.includes(COPYRIGHT)) {
-    console.error(`Missing/incorrect copyright header: ${path}`);
-    if (!failed) {
-      failed = true;
-    }
+    const contentWithCopyright = COPYRIGHT + "\n" + content;
```

Comment:
Should there also be a `\r` on Windows?

Reply:
@kt3k was in favour of leaving it as `\n`.